src/runtime_src/core/tools/xbutil2/OO_AieRegRead.cpp

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (C) 2021 Licensed under the Apache License, Version
+ * 2.0 (the "License"). You may not use this file except in
+ * compliance with the License. A copy of the License is located
+ * at
```

Comment:
Is this license header correctly formatted?

Reply:
fixed